### ISSUE TITLE
Mirror to maven central

### ISSUE THREAD

**Issue by theangrydev** [bodar/yatspec#44]
As discussed in #43, the aim of this issue is to mirror the yatspec artifacts to maven central, in order to make yatspec available to those that are unable to depend on the primary repo.bodar.com repository.

To meet the central [requirements](http://central.sonatype.org/pages/requirements.html), there are a few pieces that are still missing though:
* Creating a [New Project](https://issues.sonatype.org/secure/CreateIssue.jspa?issuetype=21&pid=10134) ticket on issues.sonatype.org
* Producing a Javadoc JAR
* Signing all artifacts with a GPG key

**Comment by wojciechbulaty** [comment]
+1

We cannot use Yatspec at a bank now. All maven artifacts are downloaded via proxying them through internal artifactory from maven central. Adding a new repo to artifactory is not possible. Downloading jar from the internet is blocked.

@danielbodart Could you add yatspec artifacts to maven central, please?

**Comment by trafficparrot** [comment]
Hey @danielbodart 

@theangrydev and @wojciechbulaty are both part of @trafficparrot. We would be happy to take the ownership of the repo under the @trafficparrot umbrella.

If that works for you, we suggest transferring the yatspec repository to @trafficparrot https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/

A transfer is transparent to existing users and contributors, because the redirects will be handled automatically by github.
A fork works for us as well, in that case we would fork and update bodar/yatspec/README to point at trafficparrot/yatspec.
Adding @theangrydev and @wojciechbulaty as admins to bodar/yatspec is also an option, though not a preference for us. Our preferred option is to take yatspec under the @trafficparrot umbrella by doing a repository transfer.

Does that work for you? If so, which option do you prefer?

**Comment by theangrydev** [comment]
Personally I think a fork is more appropriate than a transfer. We do not yet know exactly which future direction we will take with the fork and so to avoid upsetting existing users it is probably best to keep vanilla yatspec as it is and let people migrate to the fork if they wish to. A new name for the forked project would make sense too to avoid confusion with vanilla yatspec.

**Comment by trafficparrot** [comment]
@theangrydev Agreed. In that case, I would link from bodar/yatspec/README to trafficparrot/thenewprojectname, as long as @danielbodart is fine with it.

P.S.
We have some improvements to the documentation we wanna make, so we can add them in both bodar/yatspec and trafficparrot/thenewprojectname.
